common/persistence/persistenceMetricClients.go

```diff
@@ -224,7 +224,8 @@ func NewConfigStorePersistenceMetricsClient(
 	}
 }
 
-func (p *persistenceMetricsClientBase) updateErrorMetric(scope int, err error) {
+func (p *persistenceMetricsClientBase) updateErrorMetric(scope int, err error, scopeWithDomainTag ...metrics.Scope) {
+
```

Comment:
It seems that you added a new parameter, but didn't use the parameter in this function.

Reply:
Yes, I had a doubt regarding in the function: whether to put everything in if/else as per domainName is coming or not?